src/pages/Home/components/SectionDirection.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './SectionDirection.scss';
+
+function getClassesByDirection({ direction }) {
+  if (direction === 'left') {
```

Comment:
`return direction === 'left' ? styles.hasLeftDirection : styles.hasRightDirection;` But again... try to use CSS pseudo-classes instead of a variable.

Reply:
Mhh.. If you mean to use the pseudo-classes to create the order like:
Since **About is first** and **Mission is second**
`:nth-child(1) ---> make this flex-direction: row`
`:nth-child(2) ---> make this flex-direction: row-reverse`

I don't agree with that... Because you are editing the behaviour of the component from another point of the application. What if now instead of:
```
<Home>
    <About/>
    <Mission/>
</Home>
```
We have:
```
<Home>
    <Mission/>
    <About/>
</Home>
```

The components would behave different from what we expect **because now:**
**Mision is** `nth-child(1)`
**About is** `nth-child(2)`

But, what we can do is to add a `classname` to each instance, but then, we would have to add a new modifier to each new instance. It means that our `.scss` could grow as more instances we have

Reply:
I don't understand why you're talking about changing the order. 
![image](https://user-images.githubusercontent.com/7671983/45940610-5682f980-bfda-11e8-975e-a0b54bdfcdf5.png).

About & contact have the same layout (text right, images left).  Mission has the opposite. 

Is there a reason we can't use nth-child(even) to display the about and contact pages right.... I don't want to have a billion class names added everywhere for simple layout issues which can be fixed with pseudo classes. Styling with nth-child has nothing to do with DOM structure; just display.

Reply:
Use the pseudo-classes to style different/alternating layouts. Do not use left/right classes on everything.

Reply:
No, you shouldn't repeat it. Create a mixin or something that we can include. Or place this in a Sass partial which extends to all components.